packages/css/avatar.css

```diff
@@ -84,14 +85,14 @@
   }
 
   &[data-size='sm'] {
-    @composes ds-heading-text--2xs from './base/base.css';
+    font-size: var(--ds-heading-2xs-font-size);
   }
 
   &[data-size='md'] {
-    @composes ds-heading-text--sm from './base/base.css';
+    font-size: var(--ds-heading-sm-font-size);
   }
 
   &[data-size='lg'] {
-    @composes ds-heading-text--md from './base/base.css';
+    font-size: var(--ds-heading-md-font-size);
   }
```

Comment:
Don't we need `letter-spacing` and `font-weight` here?

Reply:
Thats implying the Avatar initials should have the Heading styling as well, but design was not sure about this - font sizes was mostly used to scale with the shape. Lasse and I want to include Øyvind on this one, so it is in the todo list for https://github.com/digdir/designsystemet/issues/2665, but most likely we'll disconnect from heading sizes completely and rather use container query to scale font to match size :)

Reply:
Sure, but I think we should set something here, otherwise it will inherit and avatars may look different depending on where you put it.

Reply:
Thats intentional; if letter-spacing is scaled up on body-text, this should be inherited as this is probably done as an adjustment to the currently used font. Font-weight is set on line 15 in `avatar.css` and line-height is redundant as we are using `display: flex` and initials will always be single line :)